<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowBase</class>
 <widget class="QMainWindow" name="MainWindowBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="leftFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QMdiArea" name="mdiArea"/>
      <widget class="QFrame" name="rightFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionProjectNew"/>
    <addaction name="actionProjectOpen"/>
    <addaction name="separator"/>
    <addaction name="actionProjectSave"/>
    <addaction name="actionProjectSaveAs"/>
    <addaction name="actionProjectClose"/>
    <addaction name="separator"/>
    <addaction name="actionAppClose"/>
   </widget>
   <widget class="QMenu" name="menu_Windows">
    <property name="title">
     <string>&amp;Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionLoggerToggle"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuRendering"/>
   <addaction name="menu_Windows"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionProjectNew"/>
   <addaction name="actionProjectOpen"/>
   <addaction name="actionProjectSave"/>
   <addaction name="separator"/>
   <addaction name="actionFullScreen"/>
   <addaction name="separator"/>
   <addaction name="actionSaveScreenshot"/>
   <addaction name="actionLoggerToggle"/>
   <addaction name="actionPreferences"/>
  </widget>
  <widget class="QDockWidget" name="statusDockWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>System Log Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="wombat::QMessageWriter" name="messageWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionProjectOpen">
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/document-open.png</normaloff>:/icons/24/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;open project</string>
   </property>
   <property name="statusTip">
    <string>open a project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionProjectSave">
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/document-save.png</normaloff>:/icons/24/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;save project</string>
   </property>
   <property name="statusTip">
    <string>save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionProjectClose">
   <property name="text">
    <string>close project</string>
   </property>
   <property name="statusTip">
    <string>close project</string>
   </property>
  </action>
  <action name="actionAppClose">
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/system-shutdown.png</normaloff>:/icons/24/system-shutdown.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
   <property name="statusTip">
    <string>close application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoggerToggle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/dialog-warning.png</normaloff>:/icons/24/dialog-warning.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Logger</string>
   </property>
   <property name="statusTip">
    <string>toggle logger visibility</string>
   </property>
  </action>
  <action name="actionProjectSaveAs">
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/document-save-as.png</normaloff>:/icons/24/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>save project as ..</string>
   </property>
   <property name="statusTip">
    <string>save project as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/preferences-system-session.png</normaloff>:/icons/24/preferences-system-session.png</iconset>
   </property>
   <property name="text">
    <string>preferences</string>
   </property>
   <property name="statusTip">
    <string>show preferences</string>
   </property>
  </action>
  <action name="actionProjectNew">
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/document-new.png</normaloff>:/icons/24/document-new.png</iconset>
   </property>
   <property name="text">
    <string>new project</string>
   </property>
   <property name="statusTip">
    <string>create new project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/view-fullscreen.png</normaloff>:/icons/24/view-fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>full screen</string>
   </property>
   <property name="toolTip">
    <string>show fullscreen</string>
   </property>
   <property name="statusTip">
    <string>show application fullscreen</string>
   </property>
  </action>
  <action name="actionSaveScreenshot">
   <property name="icon">
    <iconset resource="../../lib/wombatui_qt/resources/wombat.qrc">
     <normaloff>:/icons/24/photo.png</normaloff>:/icons/24/photo.png</iconset>
   </property>
   <property name="text">
    <string>Save Screenshot</string>
   </property>
   <property name="toolTip">
    <string>save a screenshot taken from the primary render widget</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wombat::QMessageWriter</class>
   <extends>QListWidget</extends>
   <header location="global">wombatui_qt/helpers/QMessageWriter.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../lib/wombatui_qt/resources/wombat.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionLoggerToggle</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusDockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
